src/styles/styles.js

```diff
@@ -494,6 +548,68 @@ const styles = {
         textAlignVertical: 'center',
     },
 
+    expensiPickerContainer: {
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
+        borderColor: themeColors.border,
+        justifyContent: 'center',
+        backgroundColor: '#fff',
+    },
+    expensiPickerContainerOnFocus: {
```

Comment:
This style is another exact duplicate of `expensiTextInputContainerOnFocus`. They should both just be something generic and reusable like `borderColorFocus`

Reply:
Done